NTumbleBit/ClassicTumbler/Client/TumblerClient.cs

```diff
@@ -154,17 +156,29 @@ public TimeSpan RequestTimeout
 				string error = await result.Content.ReadAsStringAsync().ConfigureAwait(false);
 				if(!string.IsNullOrEmpty(error))
 				{
-					throw new HttpRequestException(result.StatusCode + ": " + error);
+                    throw new HttpRequestException(result.StatusCode + ": " + error);
 				}
 			}
 			if(result.StatusCode == HttpStatusCode.NotFound)
 				return default(T);
-			if(result.Content?.Headers?.ContentLength > MaxContentLength)
-				throw new IOException("Content is too big");
-
-			result.EnsureSuccessStatusCode();
-			if(typeof(T) == typeof(byte[]))
-				return (T)(object)await result.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
+            if (result.Content?.Headers?.ContentLength > MaxContentLength)
+            {
+                Logs.Client.LogError($"Content is larger than max content lenth of {MaxContentLength}bytes for request to: {uri}\nRequest Body: \"{body.ToString()}\"");
```

Comment:
More logging can't hurt, but do we see this error occurring in practice?

Reply:
I've never seen an exception thrown but I had exception being thrown from this method and thought to improve messages for all cases.

Reply:
@D3m0nKingx Fixed the typo